titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/validator/JobImageValidatorConfiguration.java

```diff
@@ -23,4 +23,11 @@
 public interface JobImageValidatorConfiguration {
     @DefaultValue("true")
     boolean isEnabled();
+
+    /**
+     * Since Image validations are on the job accept path the timeout value is aggressive.
+     * This must be smaller than {@link TitusValidatorConfiguration#getTimeoutMs()}.
+     */
+    @DefaultValue("1000")
```

Comment:
It might also be worth adjusting the retry count in `TitusRegistryClientConfiguration` because this would timeout while the `DefaultDockerRegistryClient` is still still retrying. Maybe 2 500ms retries at the registry client and a 1500ms timeout here?

Reply:
That could potentially only leave 500ms for the other sanitizers (Iam) to run. How about 2x 400ms? Is there anything experimental about the 500ms timeout?

Reply:
We've been using the 500ms for sometime now, so the 400ms would be new. I think we lost latency metrics though going Rx->Spring :-( FWIW, the IAM sanitization should be fast as the lookup should effectively always be cached since validation run just prior to sanitization adds it to the cache. But I'm OK using either 400 or 500ms.

Reply:
alright, I'll change this to 1500ms and tweak the `TitusRegistryClientConfiguration`